src/nbl/asset/interchange/CGLTFLoader.cpp

```diff
@@ -324,7 +324,7 @@ using namespace nbl::asset;
 				{
 					auto desc = material.descriptorSet->getDescriptors(i).begin();
 					desc->desc = defaultImageView;
-					desc->image.imageLayout = EIL_SHADER_READ_ONLY_OPTIMAL;
+					desc->image.imageLayout = IImage::EL_SHADER_READ_ONLY_OPTIMAL;
```

Comment:
Should I test if GLTF example works after these changes? or are these trivial?

Reply:
I consider the GLTF loader WIP and broken ;)